### DISCUSSION TITLE
Documentation error

### DISCUSSION THREAD

**Discussion by KyleC69** [AvaloniaUI/Avalonia#12007] [General]
Following the title How to create a custom control the second page (Next button) is linked to a completely different page unrelated to topic. Occurs under the 11.0 version doc pages.

**Comment by timunie** [comment]
do you think you can provide a PR to fix it? Your contribution will be welcome.

**Comment by MikeCodesDotNET** [comment]
The Page: https://docs.avaloniaui.net/docs/next/guides/custom-controls/
The Next Page: https://docs.avaloniaui.net/docs/next/guides/custom-controls/types-of-control

Probably need to take a look at what was in the original flow. The titles don't match, which makes me think there has been a mistake.

**Comment by MikeCodesDotNET** [comment]
I think I've [resolved it](https://github.com/AvaloniaUI/avalonia-docs/commit/7769291c176228716ecb87e8c4243008aa3b38ba).
